tool/teleport/common/teleport.go

```diff
@@ -1017,6 +1022,27 @@ func onIntegrationConfEICEIAM(params config.IntegrationConfEICEIAM) error {
 	return nil
 }
 
+func onIntegrationConfAWSAppAccessIAM(params config.IntegrationConfAWSAppAccessIAM) error {
+	ctx := context.Background()
```

Comment:
Take the context as a parameter from Run?

Reply:
Run has no `context` so far, but I'll create one :+1:

Reply:
I'm surprised it went without a context this long.